doc/meson.build

```diff
@@ -0,0 +1,57 @@
+c2man = find_program('c2man', required: false)
+
+if not c2man.found()
+  error('c2man is required to build documentation. Or disable with -Ddocs=false')
+endif
+
+gen_man_pages = [
+  'fribidi_charset_to_unicode.3',
+  'fribidi_debug_status.3',
+  'fribidi_get_bidi_type.3',
+  'fribidi_get_bidi_type_name.3',
+  'fribidi_get_bidi_types.3',
+  'fribidi_get_bracket.3',
+  'fribidi_get_bracket_types.3',
+  'fribidi_get_joining_type.3',
+  'fribidi_get_joining_type_name.3',
+  'fribidi_get_joining_types.3',
+  'fribidi_get_mirror_char.3',
+  'fribidi_get_par_direction.3',
+  'fribidi_get_par_embedding_levels.3',
+  'fribidi_get_par_embedding_levels_ex.3',
+  'fribidi_get_type.3',
+  'fribidi_get_type_internal.3',
+  'fribidi_join_arabic.3',
+  'fribidi_log2vis.3',
+  'fribidi_log2vis_get_embedding_levels.3',
+  'fribidi_mirroring_status.3',
+  'fribidi_parse_charset.3',
+  'fribidi_remove_bidi_marks.3',
+  'fribidi_reorder_line.3',
+  'fribidi_reorder_nsm_status.3',
+  'fribidi_set_debug.3',
+  'fribidi_set_mirroring.3',
+  'fribidi_set_reorder_nsm.3',
+  'fribidi_shape.3',
+  'fribidi_shape_arabic.3',
+  'fribidi_shape_mirroring.3',
+  'fribidi_unicode_to_charset.3',
```

Comment:
This hard-codes man pages while autotools takes it from `lib/fribidi.def` and I don’t think we want this duplication.

Reply:
Fixed.